src/grunt-rollup/tasks/rollup.js

```diff
@@ -0,0 +1,243 @@
+const path = require("path");
```

Comment:
Can you please add here a bit documentation about this. Maybe also a reference to this issue.